autotest/client/go/goclienttest/TestCache/configCacheDisable.yaml

```diff
@@ -1,5 +1,5 @@
 #singletonServer: https://localhost:8090
-singletonServer: https://localhost:8090
+singletonServer: https://g11n-vip-dev-1.eng.vmware.com:8090
```

Comment:
It seems the server can't be accessed via public network, please confirm.

Reply:
This is a configuration file that is no longer in use,  I will going to delete it